portfolio/src/main/webapp/favoriteFoods.css

```diff
@@ -0,0 +1,68 @@
+.food-container {
+    margin-top: 10px;
+    height: 100%;
+    width: 100%;
+}
+
+.columns {
+    width:20%; 
+    height:100%;
+    float:left; 
+    position:absolute; 
+    -moz-transition:all 0.5s linear;
+    border: 0.5px solid black;
+    border-radius: 5px;
+}
+
+.columns:hover {
+    width:50%;   
+    overflow:visible; 
+    z-index:1;
+}
+
+.col1 {
```

Comment:
You could create a single class column which holds the shared properties between each column, and more specific classes for each individual column

```css
.column {
  background-repeat: no-repeat;
  background-size: 100% 100%;
  transition-duration: 2s;
}

.column.col1 {
  background-image: url(/images/cachapa.jpg);
}

.column.col2 {
  background-image: url(/images/mangu.jpg);
  left: 20%;
}

Reply:
Why repeat .column in ".column.col1" instead of just stating ".col1"? I know it's accessing the same thing but does it have an effect on priority or something?

Reply:
It doesn't change anything in this case. It would make the .colN rules specific only to an element which already has .column class. Helps make things a bit more clear, but not strictly necessary. If you have conflicting properties, the most specific one will override, as well.